vessel-runtime/src/main/java/com/textnow/android/vessel/VesselCacheImpl.kt

```diff
@@ -0,0 +1,75 @@
+package com.textnow.android.vessel
+
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentLinkedDeque
+
+open class DefaultVesselCache : VesselCache {
```

Comment:
Maybe add a comment explaining what each of these cache implementations do here,